connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoReader.scala

```diff
@@ -44,9 +45,9 @@ private[kusto] object KustoReader {
   private val myName = this.getClass.getSimpleName
 
   private[kusto] def leanBuildScan(
-    kustoClient: Client,
-    request: KustoReadRequest,
-    filtering: KustoFiltering): RDD[Row] = {
+                                    kustoClient: Client,
```

Comment:
indentation?

Reply:
this is our indentation in the project. maybe first param should be in same line though